<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SketcherGui::TaskSketcherGeneral</class>
 <widget class="QWidget" name="SketcherGui::TaskSketcherGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Gui::PrefCheckBox" name="checkBoxShowGrid">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>A grid will be shown</string>
     </property>
     <property name="text">
      <string>Show grid</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="prefEntry" stdset="0">
      <cstring>ShowGrid</cstring>
     </property>
     <property name="prefPath" stdset="0">
      <cstring>Mod/Sketcher/General</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Grid size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Gui::PrefQuantitySpinBox" name="gridSize" native="true">
       <property name="toolTip">
        <string>Distance between two subsequent grid lines</string>
       </property>
       <property name="unit" stdset="0">
        <string notr="true">mm</string>
       </property>
       <property name="decimals" stdset="0">
        <number>3</number>
       </property>
       <property name="maximum" stdset="0">
        <double>99999999.000000000000000</double>
       </property>
       <property name="minimum" stdset="0">
        <double>0.001000000000000</double>
       </property>
       <property name="singleStep" stdset="0">
        <double>1.000000000000000</double>
       </property>
       <property name="value" stdset="0">
        <double>10.000000000000000</double>
       </property>
       <property name="prefEntry" stdset="0">
        <cstring>GridSize</cstring>
       </property>
       <property name="prefPath" stdset="0">
        <cstring>Mod/Sketcher/General/GridSize</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="Gui::PrefCheckBox" name="checkBoxGridSnap">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>New points will snap to the nearest grid line.
Points must be set closer than a fifth of the grid size to a grid line to snap.</string>
       </property>
       <property name="text">
        <string>Grid snap</string>
       </property>
       <property name="prefEntry" stdset="0">
        <cstring>GridSnap</cstring>
       </property>
       <property name="prefPath" stdset="0">
        <cstring>Mod/Sketcher/General</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxGridAutoScale">
       <property name="toolTip">
        <string>Auto change grid size depending on the zoom level of the current view.</string>
       </property>
       <property name="text">
        <string>Grid auto scale</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Gui::PrefCheckBox" name="showOriginalColor">
     <property name="toolTip">
      <string>Do not change color for invalid or fully constrained sketch.</string>
     </property>
     <property name="text">
      <string>Show original color</string>
     </property>
     <property name="prefPath" stdset="0">
      <cstring>Mod/Sketcher/General</cstring>
     </property>
     <property name="prefEntry" stdset="0">
      <cstring>ShowOriginalColor</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Gui::PrefCheckBox" name="checkBoxAllowFace">
     <property name="text">
      <string>Enable face external geometry picking</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="prefEntry" stdset="0">
      <cstring>AllowFaceExternalPick</cstring>
     </property>
     <property name="prefPath" stdset="0">
      <cstring>Mod/Sketcher/General</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Gui::PrefCheckBox" name="autoTransparentPick">
     <property name="toolTip">
      <string>Auto make pre-selected shape transparent for picking
hidden edges and points on picking external objects.</string>
     </property>
     <property name="text">
      <string>Auto transparent on picking</string>
     </property>
     <property name="prefPath" stdset="0">
      <cstring>Mod/Sketcher/General</cstring>
     </property>
     <property name="prefEntry" stdset="0">
      <cstring>SketchAutoTransparentPick</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Gui::PrefCheckBox" name="checkBoxAutoconstraints">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Sketcher proposes automatically sensible constraints.</string>
     </property>
     <property name="text">
      <string>Auto constraints</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="prefEntry" stdset="0">
      <cstring>AutoConstraints</cstring>
     </property>
     <property name="prefPath" stdset="0">
      <cstring>Mod/Sketcher/General</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Gui::PrefCheckBox" name="checkBoxRedundantAutoconstraints">
     <property name="toolTip">
      <string>Sketcher tries not to propose redundant auto constraints</string>
     </property>
     <property name="text">
      <string>Avoid redundant auto constraints</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="prefEntry" stdset="0">
      <cstring>AvoidRedundantAutoconstraints</cstring>
     </property>
     <property name="prefPath" stdset="0">
      <cstring>Mod/Sketcher/General</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Rendering order (global):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="renderingOrder">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>12</width>
       <height>60</height>
      </size>
     </property>
     <property name="toolTip">
      <string>To change, drag and drop a geometry type to top or bottom</string>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Fixed</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::PrefCheckBox</class>
   <extends>QCheckBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::PrefQuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
